quinn-proto/src/connection/spaces.rs

```diff
@@ -437,6 +437,8 @@ impl SendableFrames {
 pub(crate) struct PendingAcks {
     permit_ack_only: bool,
     ranges: ArrayRangeSet,
+    latest_incoming: Option<Instant>,
```

Comment:
Let's leave a comment here explaining the intended future application, since it's otherwise mysteriously unused.

Reply:
done